index.html

```diff
@@ -1182,13 +1187,19 @@ <h4>
               <ol>
                 <li>Resolve <var>U</var> relative to the API base URL specified
                 by the <a>current settings object</a>, and add the resulting
-                absolute URL (if any) to <var>presentationUrls</var>.
+                absolute URL (if any, and if the resulting scheme is supported
```

Comment:
This might be easier to read if it were broken into two steps.  Here is one way that could be done.

1. Let A be absolute URL that is the result of resolving <var>U</var> relative to the API base URL specified by the <a>current settings object</a>.
2. If the <a>parse a URL</a> algorithm failed, then throw a <a>SyntaxError</a> exception and abort all remaining steps.
3. If A has a scheme supported by the controlling user agent, add A to presentationUrls.

Reply:
LGTM. Thanks for your suggestion.